packages/cli/src/databases/entities/api-key.ts

```diff
@@ -0,0 +1,25 @@
+import { Column, Entity, Index, ManyToOne, Unique } from '@n8n/typeorm';
+
+import { WithTimestampsAndStringId } from './abstract-entity';
+import { User } from './user';
+
+@Entity('user_api_keys')
```

Comment:
We seem to have mix of tables in plural and singular. Should unify the table names in the future

Reply:
Yesssss! we need to do this. However, how can en enforce the names after the names? 🤔

Reply:
Probably need a lint rule and/or test